mso/resource_mso_schema_site_anp_epg_domain.go

```diff
@@ -309,36 +362,71 @@ func resourceMSOSchemaSiteAnpEpgDomainCreate(d *schema.ResourceData, m interface
 	anpName := d.Get("anp_name").(string)
 	epgName := d.Get("epg_name").(string)
 	domainType := d.Get("domain_type").(string)
-	domainName := d.Get("dn").(string)
+	vmmDomainType := d.Get("vmm_domain_type").(string)
+	domainNameDnOld := d.Get("dn").(string)
```

Comment:
Why doing the get here and a getOk later? just do the getOk like the others.

Reply:
ok